content/01.abstract.md

```diff
@@ -1,11 +1,10 @@
 ## Abstract {.page_break_before}
 
-Deep learning, which describes a class of machine learning algorithms, has recently showed impressive results across a variety of domains.
-Biology and medicine are data rich, but the data are complex and often ill-understood.
-Problems of this nature may be particularly well-suited to deep learning techniques.
-We examine applications of deep learning to a variety of biomedical problems---patient classification, fundamental biological processes, and treatment of patients---and discuss whether deep learning will transform these tasks or if the biomedical sphere poses unique challenges.
-We find that deep learning has yet to revolutionize or definitively resolve any of these problems, but promising advances have been made on the prior state of the art.
-Even when improvement over a previous baseline has been modest, we have seen signs that deep learning methods may speed or aid human investigation.
-More work is needed to address concerns related to interpretability and how to best model each problem.
+Deep learning, which describes a class of machine learning algorithms focussing on the training of deep artificial neural networks, has recently shown impressive results across a variety of domains.
+Biology and medicine are data-rich disciplines, but the data are complex and often ill-understood. Hence, deep learning techniques may be particularly well-suited to solve problems of these fields.
```

Comment:
I don't really like this (or our initial framing). What about:

> Deep learning techniques may be particularly well suited for challenges in biology and medicine, which are data-rich disciplines where there is often a complex biological system between what we can measure and what we wish to know.

Reply:
☝️ needs revision too, but I vaguely like the direction it's going in

Reply:
I'm okay with the @rasbt version here but am happy to switch it if you want.